test/modules/utils.js

```diff
@@ -3,6 +3,15 @@
 // Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Why move these values and functions to `./utils` from `./environment`?
In be able to import them without also engaging ChromeDriver env which is tied to `environment.js`.